tests/spec/core/biblio-spec.js

```diff
@@ -209,3 +209,37 @@ describe("W3C — Bibliographic References", () => {
     expect(refs[0].textContent).toEqual("[dom]");
   });
 });
+
+it("makes sure the Informative references section has expected localization text", async () => {
+  const ops = {
+    config: makeBasicConfig(),
+    htmlAttrs: {
+      lang: "nl",
+    },
+    body: `<section class="informative" id="intro">[[DOM]]</section>`,
+  };
+  const doc = await makeRSDoc(ops);
+  const ref = doc.getElementById("references");
+  expect(doc.documentElement.lang).toEqual("nl");
+  expect(ref.querySelector("h2").textContent).toMatch("Referenties");
```

Comment:
```suggestion
  expect(textContent.trim()).toBe("Referenties");
```

Reply:
@marcoscaceres I've made requested changes. However I used `substr()` to remove space and unwanted characters from `textContent` as `trim()` only removed white space and caused test to fail.